dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityService.java

```diff
@@ -248,12 +248,9 @@ public TrackedEntity getTrackedEntity(
    *
    * @return the TE object if found and accessible by the current user
    * @throws NotFoundException if uid does not exist
-   * @throws ForbiddenException if TE registration org unit not in user's scope or not enough
-   *     sharing access
    */
   private TrackedEntity getTrackedEntity(
-      String uid, TrackedEntityParams params, boolean includeDeleted)
-      throws NotFoundException, ForbiddenException {
+      String uid, TrackedEntityParams params, boolean includeDeleted) throws NotFoundException {
     TrackedEntity trackedEntity = trackedEntityStore.getByUid(uid);
     addTrackedEntityAudit(trackedEntity, CurrentUserUtil.getCurrentUsername());
     if (trackedEntity == null) {
```

Comment:
@enricocolasante should I also return null instead of throwing a `NotFoundException` here?